package.json

```diff
@@ -58,6 +58,7 @@
     "bcrypt": "^3.0.6",
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.18.2",
+    "chance": "^1.0.18",
```

Comment:
If you are using chance directly from the `factory` then this is not needed, but uninstall it for next PR.